dvc/fs/dvc.py

```diff
@@ -79,6 +79,11 @@ def _merge_info(repo, fs_info, dvc_info):
     return ret
 
 
+def _get_dvc_path(dvc_fs, dvc_parts):
+    if dvc_fs:
```

Comment:
Seems like in both uses we already check that dvc_fs is not none

Reply:
`_get_fs_pair()` doesn't check (indeed, I moved the check from there) but it seems that `dvc_fs` is never None anyway.